src/js/components/PodContainerSpecView.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+import DescriptionList from './DescriptionList';
+import MarathonConfigUtil from '../utils/MarathonConfigUtil';
+
+const METHODS_TO_BIND = [
+];
+
+class PodContainerSpecView extends React.Component {
+  constructor() {
+    super(...arguments);
+
+    this.state = {
```

Comment:
Lets remove `state` declarations since this is a stateless component. Also don't need anything in the constructor so we could just remove the constructor here.